plugins/inputs/nvidia_smi/nvidia_smi.go

```diff
@@ -119,8 +126,10 @@ func (smi *NvidiaSMI) parse(acc telegraf.Accumulator, data []byte) error {
 func init() {
 	inputs.Add("nvidia_smi", func() telegraf.Input {
 		return &NvidiaSMI{
-			BinPath: "/usr/bin/nvidia-smi",
-			Timeout: config.Duration(5 * time.Second),
+			BinPath:       "/usr/bin/nvidia-smi",
+			Timeout:       config.Duration(5 * time.Second),
+			TestOnStartup: false,
+			nvidiaSMIArgs: []string{"-q", "-x"},
```

Comment:
Please do not set options to their golang defaults!
```suggestion
			nvidiaSMIArgs: []string{"-q", "-x"},
```
Furthermore, I  would prefer setting the args in the `Init()` function and use a less bulky name like `args` or `cmdargs`...